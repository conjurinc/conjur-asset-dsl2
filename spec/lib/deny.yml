- !deny
  resource: !variable foobar
  privilege: read
  member: !group developers

- !deny
  resource: !variable baz
  privilege: [ read, execute ]
  member: !member
    role: !group developers

- !deny
  resource: !variable db_password
  privilege: read
  member:
    role: !group developers
