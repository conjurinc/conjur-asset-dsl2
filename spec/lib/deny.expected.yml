---
- !deny
  resource: !variable
    id: foobar
  privilege: read
  member: !member
    role: !group
      id: developers
- !deny
  resource: !variable
    id: baz
  privilege:
  - read
  - execute
  member: !member
    role: !group
      id: developers
- !deny
  resource: !variable
    id: db_password
  privilege: read
  member: !member
    role: !group
      id: developers
